tools/cgpatch/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+project(CGPatch)
+
+# LLVM
+if(NOT LLVM_ENABLE_RTTI)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-rtti")
+endif()
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
```

Comment:
This should probably use `target_compile_options` to set this flag instead of the global CMake flags var